get-resource.sh

```diff
@@ -63,6 +63,13 @@ else
       unxz "$RHCOS_IMAGE_FILENAME_RAW"
     fi
 
+    if [[ $API_VIP =~ : ]]; then
```

Comment:
Can you add a comment here explaining why we do this? And also the commit message should say something useful.

Reply:
and I think that in the case of dual_stack we need to do nothing, otherwise it will force the node to have only ipv4 or ipv6